.github/workflows/prepare-custom-action.yml

```diff
@@ -0,0 +1,51 @@
+name: prepare-custom-action
+
+on:
+  schedule:
+    - cron: '0 0 * * 5'
+
+  push:
+    branches: [ "main" ]
+    paths:
+      - "Dockerfile"
+      - "entrypoint.sh"
+    
+  workflow_dispatch:
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    permissions:
+      contents: read
+      packages: write
+    defaults:
+      run:
+        shell: bash
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@65b78e6e13532edd9afa3aa52ac7964289d1a9c1
```

Comment:
Although I have had a similar intention, since we publish to GitHub Packages, I have taken GitHub's documentation's recommended flow (since each container could have its details). I would favor keeping the hash per documentation if there are no particular known issues with these that might impact us. https://docs.github.com/en/actions/publishing-packages/publishing-docker-images

Reply:
No strong feeling here but I do somewhat prefer versioned releases as well. In particular because if we _do_ run into an issue, the hash makes it hard to find out whether we're behind or not. Unfortunately this hash doesn't correspond to a version, but it's between v2.1.0 and v2.2.0. v.2.2.0 is the latest release.

Reply:
Migrated all actions to versioned, and it works! Thanks a lot!